README.md

```diff
@@ -1,5 +1,6 @@
 # GitHub For Dummies Readers
 This is a repository where all GitHub For Dummies readers can add a link to their GitHub profile!
+Usre!
```

Comment:
👋 I think instead of adding something here, you can add something to the end of the table below 😄

```suggestion
```